drivers/volume/portworx/schedops/schedops.go

```diff
@@ -29,6 +29,8 @@ type Driver interface {
 	GetServiceEndpoint() (string, error)
 	// UpgradePortworx upgrades portworx to the given docker image and tag
 	UpgradePortworx(image, tag string) error
+	//IsPXPodRunningOnNode returns true if PX pod is up on that node, else returns false
+	IsPXPodRunningOnNode(n node.Node) bool
```

Comment:
nit: let's not have "Pod" in this method name as it is a scheduler specific name.
Up to you, we can change this later as well when we support other schedulers.

Reply:
Changed to "IsPXAppRunningOnNode".